esp32/modules/splash.py

```diff
@@ -269,30 +267,10 @@ def splash_input_init():
     ugfx.input_attach(ugfx.JOY_LEFT, splash_input_other)
     ugfx.input_attach(ugfx.JOY_RIGHT, splash_input_other)
 
-# Event timer
-def splash_timer_init():
-    global splashTimer
-    try:
-        splashTimer
-        print("[SPLASH] Timer exists already")
-    except:
-        splashTimer = machine.Timer(-1)
-        splashTimer.init(period=badge.nvs_get_u16('splash', 'timer.period', 100), mode=machine.Timer.ONE_SHOT, callback=splash_timer_callback)
-        print("[SPLASH] Timer created")
-
-def splash_timer_callback(tmr):
-    try:
-        if services.loop(splash_power_countdown_get()):
-            splash_power_countdown_reset()
-    except:
-        pass
-    if not splash_power_countdown_trigger():
-        splash_draw(False, False)
-    tmr.init(period=badge.nvs_get_u16('splash', 'timer.period', 100), mode=machine.Timer.ONE_SHOT, callback=splash_timer_callback)
-
-
 ### PROGRAM
 
+powerTimer = machine.Timer(2) #TODO: how to get this number?!?
```

Comment:
No, for now it works. I just have to keep track of the numbers I use. Currently 0 and 1 are in use by the services.py and 2 is in use by splash.py

it's ugly and I want a fix, but that fix is not needed to get it to work.